dataland-frontend/src/utils/FormKitPlugins.ts

```diff
@@ -14,6 +14,17 @@ export function selectNothingIfNotExistsFormKitPlugin(node: FormKitNode): void {
   });
 }
 
+/**
```

Comment:
What does this do? Is this connected to bein able to remove an input from something like a radio button?

Reply:
@MarcinStanczyk7  do you know something about this?

Reply:
There is no diff-to-main in this file anymore. Therefore you can close this comment @fsoffel